go.mod

```diff
@@ -2,20 +2,21 @@ module github.com/crossplaneio/crossplane
 
 go 1.12
 
+replace github.com/crossplaneio/crossplane-runtime => github.com/muvaf/crossplane-runtime v0.0.0-20191206111112-1fe048748dc5
```

Comment:
I've merged the crossplane-runtime PR. Let me know when this replace statement is removed and I'll merge.

Reply:
@negz done!